lib/fs.js

```diff
@@ -286,6 +286,7 @@ function existsSync(path) {
   try {
     path = getValidatedPath(path);
   } catch {
+    util.deprecate(() => {}, 'never throw on invalid arguments for fs.existsSync is deprecated', 'DEP0186');
```

Comment:
This means that this is not a documentation-only deprecation.

Reply:
Fixed, I think this Runtime, isn't it?